README.md

```diff
@@ -1,14 +1,78 @@
-# Auth
+# Waterpark Server
```

Comment:
Copied over from #14 

TODO: update this to relate to auth. Not sure if that's something we want to add to this PR.